### PR TITLE
Adds Faculty page (WIP)

### PR DESCRIPTION
**Pull request by dinskid**
Have 1 bug IDK how to sort. Some images dont fill up the entire container. Tried setting css property, but still couldn't get it to work. Otherwise, all the elements are functional.

Deployed @ [iiittd.surge.sh](url). Still haven't resolved the problem in surge.. will do it asap..so for now, page has to be accessed from the navbar only

### CODE REVIEW

**Comment by fahad-israr** [comment]
i am seeing it.
Always refer to the issue you are solving in PR.
example Issue @ https://github.com/helloworld-iiitt/iiitt/issues/7

Github will show on the issue automatically that you refernced.

**Comment by fahad-israr** [comment]
AFAICS
- The image size is constrained by the container sizes we've kept.
- Either we can fix height or the width.
- So that's just happening bcoz of different image sizes(resolutions)
- Leave it as it is,will look for images of uniform sizes.
- Good work!! Finish the Staff Page as well(Quite Identical I Guess)

File: src/components/person_card/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    const showMore=(e)=>{
+        setContent(props.researchArea);
+    }
+    console.log(props.src);
+
+    return (
+        <>
+            <div className={`container`}>
+                <div className="pic">
+                    <img src={require(`../../images/${props.src}`)} alt={props.name} className="image" />
```
Comment (fahad-israr) [inline src/components/person_card/index.js:42]: Why dont we write here `/images/people/faculty` so that we have to write  image name only in JSON file ?

Comment (dinskid) [reply src/components/person_card/index.js:42]: Okay

Comment (dinskid) [reply src/components/person_card/index.js]: Made required changes

**Comment by dinskid** [comment]
Added Staff and Faculty pages (#7). (WIP) Footer is buggy (doesn't stick to bottom of page)

**Comment by fahad-israr** [comment]
@dinskid  push your changes to iiittd.surge.sh as that's where I can preview!!!
Always do that

**Comment by dinskid** [comment]
Sorry about it.. I thought I did it, but looks like I didn't. Just a warning, that page looks ugly if I use just the current components. I didn't know how to design it just having 4 elements.
As always, it is in iiittd.surge.sh

**Comment by fahad-israr** [comment]
@dinskid  Try to pull the latest changes from master branch.I've fixed the footer.
I am sure it would resolve the issue.

File: src/components/person_card/styles.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.container {
+    display: grid;
+    grid-template-columns: 1.5fr 3.5fr;
+    border: 2px solid #3f51b58e;
```
Comment (fahad-israr) [inline src/components/person_card/styles.css:4]: Change border color: simple and sobre.
```suggestion
    border: 2px solid #ececed;
```

**Comment by dinskid** [comment]
Also added the home icon against issue #9

**Pull request merged**